architecture/additional_concepts/storage.adoc

```diff
@@ -254,6 +254,64 @@ take advantage of the volume's ability for ROX. Errors in the provider show up
 at runtime as mount errors.
 ====
 
+The table below lists the access modes supported by different persistent volumes.
+
+.Supported Access Modes for Persistent Volumes
+[cols=",^v,^v,^v", width="100%",options="header"]
+|===
+|Volume Plugin  |ReadWriteOnce  |ReadOnlyMany  |ReadWriteMany  
+|AWS EBS  | X | - |  -
+|Azure Disk | X | - | - 
+|Ceph RBD  | X | X |  -
+|Fiber Channel  | X | X |  -  
+|GCE Persistent Disk  | X | - |  -
+|GlusterFS  | X | X |  X
+|HostPath  | X | - |  -
+|iSCSI  | X | X |  -
+|NFS  | X | X | X 
+|Openstack Cinder  | X | - |  -
+|===
+
+[NOTE]
+====
+ * Please use link:https://docs.openshift.com/container-platform/latest/dev_guide/deployments/deployment_strategies.html#recreate-strategy[Recreate Deployment Strategy], if pods rely on AWS EBS, GCE Persistent Disks, or Openstack Cinder PVs.
```

Comment:
@rootfs Can you please verify if AWS EBS, GCE Persistent Disks, or Openstack Cinder Volume sources are shareable across nodes. Thanks!